ERROR:root:(IntegrityError) UNIQUE constraint failed: project.data u'INSERT INTO project (name, category, data) VALUES (?, ?, ?)' ('test', 'test', '')
ERROR:root:This Session's transaction has been rolled back due to a previous exception during flush. To begin a new transaction with this Session, first issue Session.rollback(). Original exception was: (IntegrityError) UNIQUE constraint failed: project.data u'INSERT INTO project (name, category, data) VALUES (?, ?, ?)' ('test', 'test', '')
ERROR:root:This Session's transaction has been rolled back due to a previous exception during flush. To begin a new transaction with this Session, first issue Session.rollback(). Original exception was: (IntegrityError) UNIQUE constraint failed: project.data u'INSERT INTO project (name, category, data) VALUES (?, ?, ?)' ('test', 'test', '')
ERROR:root:This Session's transaction has been rolled back due to a previous exception during flush. To begin a new transaction with this Session, first issue Session.rollback(). Original exception was: (IntegrityError) UNIQUE constraint failed: project.data u'INSERT INTO project (name, category, data) VALUES (?, ?, ?)' ('test', 'test', '')
ERROR:root:This Session's transaction has been rolled back due to a previous exception during flush. To begin a new transaction with this Session, first issue Session.rollback(). Original exception was: (IntegrityError) UNIQUE constraint failed: project.data u'INSERT INTO project (name, category, data) VALUES (?, ?, ?)' ('test', 'test', '')
ERROR:root:This Session's transaction has been rolled back due to a previous exception during flush. To begin a new transaction with this Session, first issue Session.rollback(). Original exception was: (IntegrityError) UNIQUE constraint failed: project.data u'INSERT INTO project (name, category, data) VALUES (?, ?, ?)' ('test', 'test', '')
ERROR:root:This Session's transaction has been rolled back due to a previous exception during flush. To begin a new transaction with this Session, first issue Session.rollback(). Original exception was: (IntegrityError) UNIQUE constraint failed: project.data u'INSERT INTO project (name, category, data) VALUES (?, ?, ?)' ('test', 'test', '')
ERROR:root:(IntegrityError) UNIQUE constraint failed: project.data u'INSERT INTO project (name, category, data) VALUES (?, ?, ?)' ('test', 'test', '')
ERROR:root:(IntegrityError) UNIQUE constraint failed: project.data u'INSERT INTO project (name, category, data) VALUES (?, ?, ?)' ('test', 'test', '')
ERROR:root:(IntegrityError) UNIQUE constraint failed: project.data u'INSERT INTO project (name, category, data) VALUES (?, ?, ?)' ('test', 'test', '')
ERROR:root:(IntegrityError) UNIQUE constraint failed: project.data u'INSERT INTO project (name, category, data) VALUES (?, ?, ?)' ('test', 'test', '')
ERROR:root:(IntegrityError) UNIQUE constraint failed: project.data u'INSERT INTO project (name, category, data) VALUES (?, ?, ?)' ('test', 'test', '')
ERROR:root:(IntegrityError) UNIQUE constraint failed: project.category u'INSERT INTO project (name, category, data) VALUES (?, ?, ?)' ('test', 'test', '#!/usr/bin/env python \nimport socket \nimport sqlite3\nimport re\nfrom project.models import *\nfrom sqlalchemy.exc import IntegrityError\nimport logging\n\n\n#define logging \n\nlogging.basicConfig(filename="app.log", level=logging.INFO)\n\n\n#define variables for connection and buffering\nhost = \'0.0.0.0\' \nport = 9000\nbacklog = 999 * 9\nsize = 999 * 9\n\n#setup a listener\ns = socket.socket(socket.AF_INET, socket.SOCK_STREAM) \ns.bind((host,port)) \ns.listen(backlog) \n\n#loop through the recieving data and parse out project/catagory\n\ndef listener():\n\ttry:\n\t\twhile 1: \n\t\t    client, address = s.accept() \n\t\t    data = client.recv(size) \n\t\t    if data: \n\t\t        client.send(data)\n\t\t        mProject = re.search(\'^([^\\/]+)\', data)\n\t\t        project = mProject.group(0)\n\t\t        mKeyword = re.search(\'\\/(.*)/\', data)\n\t\t        keyword = mKeyword.group(1)\n\t\t\tdata = \'\\n\'.join(data.split(\'\\n\')[1:]) \n\n\t\t\t\t#update the sqlite db\n\t\t    \tproject = Project(project,keyword,data)\n\t\t    \tdb.session.add(project)\n\t\t    \tdb.session.commit()\n\t\t\t#debugging\n\t\t\tprint(data)\n\n\n\n#error handling\n\texcept Exception as e:\n\t        print("Oops an exception occurred")\n\t        logging.error(e)\n\t        db.session.rollback()\n\t        db.session.flush()\n\t        listener()\n\n\nif __name__ == \'__main__\':\n\tlistener()\n')
ERROR:root:(IntegrityError) UNIQUE constraint failed: project.name u'INSERT INTO project (name, category, data) VALUES (?, ?, ?)' ('test', 'testing', '\nStarting Nmap 6.40 ( http://nmap.org ) at 2016-09-28 20:16 UTC\nNSE: Loaded 95 scripts for scanning.\nNSE: Script Pre-scanning.\nNSE: Starting runlevel 1 (of 1) scan.\nNSE: Script Post-scanning.\nNSE: Starting runlevel 1 (of 1) scan.\nRead data files from: /usr/bin/../share/nmap\nNmap done: 0 IP addresses (0 hosts up) scanned in 0.21 seconds\n           Raw packets sent: 0 (0B) | Rcvd: 0 (0B)\n')
ERROR:root:'NoneType' object has no attribute 'group'
ERROR:root:(IntegrityError) UNIQUE constraint failed: project.category u'INSERT INTO project (name, category, data) VALUES (?, ?, ?)' ('test', 'test', '\n# resume information\nresume-index = 1767\nrate =     100.00\nrandomize-hosts = true\nseed = 8954913453807906483\nshard = 1/1\n# ADAPTER SETTINGS\nadapter = \nadapter-ip = 172.16.70.103\nadapter-mac = 00:00:00:00:00:00\nrouter-mac = 00:00:00:00:00:00\n# OUTPUT/REPORTING SETTINGS\noutput-format = unknown(0)\nshow = open,,\noutput-filename = \nrotate = 0\nrotate-dir = .\nrotate-offset = 0\nrotate-filesize = 0\npcap = \n# TARGET SELECTION (IP, PORTS, EXCLUDES)\nports = 0-65535\nrange = 172.30.1.0/24\n\ncapture = cert\nnocapture = html\nnocapture = heartbleed\n\nmin-packet = 60\n')
